cpu/stm32/include/clk/clk_conf.h

```diff
@@ -22,17 +22,6 @@
 #include "kernel_defines.h"
 #include "macros/units.h"
```

Comment:
I debated pushing these includes into the individual headers as well. Since they aren't actually used here. Thoughts? This top level header is basically just a router/mux for the real headers and contains no substance.

Reply:
I would indeed find it better to move it to the individual headers since this is where the macros are used.

Reply:
will do

Reply:
done